charts/incubator/notifiarr/questions.yaml

```diff
@@ -0,0 +1,113 @@
+# Include{groups}
+portals:
+  open:
+# Include{portalLink}
+questions:
+# Include{global}
+# Include{workload}
+# Include{workloadDeployment}
+
+# Include{replicas1}
+# Include{podSpec}
+# Include{containerMain}
+                                - variable: env
+                                  group: "App Configuration"
+                                  label: "Image Environment"
+                                  schema:
+                                    additional_attrs: true
+                                    type: dict
+                                    attrs:
+                                      - variable: DN_API_KEY
+                                        label: "DN_API_KEY"
```

Comment:
Let's update the labels to something like `DN Api Key` (or whatever DN stands for)

Reply:
Just updated these.